main.go

```diff
@@ -0,0 +1,194 @@
+// vi:set tabstop=8 shiftwidth=8 noexpandtab:
+package main
+
+import (
+	"io"
+	"fmt"
+	"flag"
+	"net/http"
+	"os"
+	"os/signal"
+	"runtime"
+	"syscall"
+	"time"
+
+	"github.com/op/go-logging"
+
+	"github.com/nocive/go-request-counter/src/counter"
+	"github.com/nocive/go-request-counter/src/storage"
+)
+
+func handleError(err error) {
+	if err != nil {
+		_, file, line, _ := runtime.Caller(1)
+		logger.Fatal(fmt.Sprintf("Line: %d\tFile: %s\nMessage: %s", line, file, err))
+		os.Exit(1)
+	}
+}
+
+func boot() {
+	var err error
+
+	logger.Info(fmt.Sprintf("booting! ttl: %dms / refresh interval: %dms / save interval: %dms\n", requestTtl, RefreshInterval, SaveInterval))
```

Comment:
in all these you can just use `Infof` all loggers generally always have a `f` variant:
https://godoc.org/github.com/op/go-logging#Logger.Infof